src/goto-instrument/contracts/contracts.cpp

```diff
@@ -1445,38 +1475,34 @@ void code_contractst::add_contract_check(
   //   assert(ensures)
   // skip: ...
 
-  // build skip so that if(nondet) can refer to it
-  goto_programt tmp_skip;
-  goto_programt::targett skip =
-    tmp_skip.add(goto_programt::make_skip(ensures.source_location()));
-
```

Comment:
Thanks for cleaning this up. We don't seem to be using this `skip` target any more.
While you're at it, could you please also update the pseudocode in the comments above to reflect the current instrumentation?